rdkit/Dbase/DbReport.py

```diff
@@ -30,7 +32,9 @@
   from rdkit.Dbase.DbConnection import DbConnect
   from rdkit.Dbase import DbInfo
   from rdkit.Reports.PDFReport import PDFReport, ReportUtils
-  import os, tempfile, sys
+  from reportlab.lib import colors
```

Comment:
By moving these out here, you cause the mere import of this module to fail if reportlab isn't installed. They were previously in the function because then you could at least import this without having the extra dependency installed.
This is a not-uncommon pattern in the RDKit and one that I think we should keep.